<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindowClass</class>
 <widget class="QWidget" name="mainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListWidget::item:selected {
    background: #87CEFA; /* 浅蓝色 */
    border-radius: 5px;
    color: black;
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>175</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(60, 120, 200, 200); /* 半透明蔚蓝色 */
</string>
   </property>
   <widget class="QLabel" name="mainAvatarlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 32px;
border: 2px solid black;  /* 可选：为头像添加边框 */
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>320</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
    background: rgba(100, 160, 220, 180);
    font-size: 14px;
    padding-left: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: rgba(100, 160, 220, 180);
    border: none;
	border-image: url(:/QQ/image/搜索 .png);
}
QPushButton:hover {
    background: rgba(80, 140, 160,255); /* 悬停时稍微深一点 */
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>45</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background: transparent; /* 让 Label 继承 Widget 的背景 */
    color: white; /* 文字颜色 */
}</string>
    </property>
    <property name="text">
     <string>admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="signatureLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background: transparent; /* 让 Label 继承 Widget 的背景 */
    color: white; /* 文字颜色 */
}</string>
    </property>
    <property name="text">
     <string>这是一次测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_1Button">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    border-image: url(:/QQ/image/关闭 (1).png);
background: transparent;
}

QPushButton:hover {
    
	background-color: red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="min_1Button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.2);
}
QPushButton {
   border-image: url(:/QQ/image/最小化_2.png);
background: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="friendAddButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover {
    background-color: rgba(255, 255, 255, 0.2);
}
QPushButton{
background: transparent;
border-image: url(:/QQ/image/添加好友.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>175</y>
     <width>350</width>
     <height>525</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>350</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(128, 128, 128);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>350</width>
      <height>465</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="messageWidget">
     <widget class="QListWidget" name="messageListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>465</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="friendWidget">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>30</height>
       </rect>
      </property>
      <widget class="QPushButton" name="friendButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>好友</string>
       </property>
      </widget>
      <widget class="QPushButton" name="groupButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>群聊</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="FriendStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>350</width>
        <height>435</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QTreeWidget" name="friendListTree">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>435</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QTreeWidget" name="groupListTree">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>350</width>
          <height>435</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="spaceWidget"/>
   </widget>
   <widget class="QPushButton" name="spaceButton">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>0</y>
      <width>117</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: gray;
    font-size: 14px;
    border: none;
    padding: 5px 10px;
}

QPushButton:hover {
    color: black;
}

QPushButton:checked {
    color: black;
    font-weight: bold;
    border-bottom: 2px solid black;
}
QPushButton {
    color: gray;
    font-size: 14px;
    border: none;
    padding: 5px 10px;
}

QPushButton:hover {
    color: black;
}

QPushButton:pressed {
    color: black;
    font-weight: bold;
    border-bottom: 2px solid black;
}
</string>
    </property>
    <property name="text">
     <string>空间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="friendListButton">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>0</y>
      <width>117</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: gray;
    font-size: 14px;
    border: none;
    padding: 5px 10px;
}

QPushButton:hover {
    color: black;
}

QPushButton:checked {
    color: black;
    font-weight: bold;
    border-bottom: 2px solid black;
}
</string>
    </property>
    <property name="text">
     <string>好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="messageButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>116</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: gray;
    font-size: 14px;
    border: none;
    padding: 5px 10px;
}

QPushButton:hover {
    color: black;
}

QPushButton:checked {
    color: black;
    font-weight: bold;
    border-bottom: 2px solid black;
}

</string>
    </property>
    <property name="text">
     <string>消息</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QQ.qrc"/>
 </resources>
 <connections/>
</ui>
